modules/shared/partials/working-with-collections.adoc

```diff
@@ -0,0 +1,38 @@
+= Working with Collections
+:nav-title: Collections
+:content-type: howto
+:page-topic-type: howto
+
+[abstract]
+The 3.0 API SDKs all work with all features of Collections and Scopes.
+
+The xref:{version-server}@server:learn:data/scopes-and-collections.adoc[Collections feature] in Couchbase Server 7.x is fully implemented in the
+3.0 API versions of the Couchbase SDKs.
+
+NOTE: When working with earlier versions (before the Developer Preview in 6.5), or with other server versions, the `defaultcollection` is used from the SDK.
```

Comment:
Earlier than 7.0 (and forget about DP ;)  )

Reply:
tweaked both comments in following commit, thanks!